### DISCUSSION TITLE
06/09 안건

### DISCUSSION THREAD

**Discussion by YuHyun-P** [prgrms-fe-devcourse/FEDC2_2JAVATAYO_Dongyoung#51] [Announcements]
### Footer에 작성할 내용
![footer](https://user-images.githubusercontent.com/96400112/172552303-3f499525-1271-46fc-b8eb-049a3b3e59c6.PNG)

✅ HARDY PLANTS

### 2JAVATAYO 레포지토리와 Slack 연동
  - 동영팀에 연결
  - 단톡에 연결
  - 각자 연결
 
✅ 채널 새로 개설해서 연결
 
### TypeScript 스터디 ?

✅ 각자 공유할 내용이 있으면 Wiki에 작성

### 다른 사용자의 프로필 검색 및 조회 권한

✅ 로그인하지 않은 사용자도 사용자 검색,  조회 권한 O

### 채널 생성
- 채널 이름, 설명 정하기
![image](https://user-images.githubusercontent.com/96400112/172673190-91322cdc-e716-4c3d-a61e-1d48769a8fbe.png)

✅ 댓글로 채널명으로 하고, 설명은 없는 걸로

**Comment by Hyevvy** [comment]
단톡 DM 말씀하시는건가요?? 
안그래도 그거 설정하려고 했는데 채널에만 된다고 해서 채널을 하나 더 파도 되는지 .. 궁금하네요 
동영팀에는 너무 많은 사람들이 있어서 알림이 계속 울리면 불편하실 거 같아서요 😮

**Comment by YuHyun-P** [comment]
채널명 변수 **front/back/ios/android/designer/guitar/ai**
